README.md

```diff
@@ -150,6 +150,10 @@ The following properties are accepted:
   The `[name]` placeholder will be replaced by the name of the function, allowing you to use it to construct the path to
   the target directory.
 
+- `displayName`
```

Comment:
As you rightly pointed out, this property is called `name` in edge functions, and it would be great if we could standardise them somehow.

I know that we already have a `name` property in zip-it-and-ship-it, but it only exists in the output object, not the input. So we could potentially accept `name` as the input and then present it as something else in the output object (like `displayName`).

Alternatively, we could even repurpose the `name` property in the output object and start populating it with the value of the `name` _input_ property (and default to the current value if there's no `name` supplied). I think we only ever use that for presentation purposes anyway, which would come with the benefit of starting to show the display name in the CLI and Netlify Build when listing functions.

What do you think? (I totally understand if you don't want to make that part of this PR, just wanted to offer these suggestions for you to consider.)

Reply:
I like the idea of standardising the properties 🤔. Maybe we should start with the input `name` instead of `displayName`. But still have the output be `displayName` as you said. I'd like to really make sure we know what the usecase is for the output `name` property right now. I could investigate that in a later moment?

Reply:
Sounds good!